example/lib/page_waveform.dart

```diff
@@ -33,29 +36,33 @@ class _PageWaveformState extends State<PageWaveform> {
   double oscillateSpeed = 0;
   List<SoundProps> notes = [];
   SoundProps? sound;
+  bool canBuild = false;
 
   @override
   void initState() {
     super.initState();
 
-    /// listen to player events
-    SoLoud.instance.audioEvent.stream.listen((event) async {
-      if (event == AudioEvent.isolateStarted) {
-        /// When it starts initialize notes
+    /// Ensure the player is down
+    if (SoLoud.instance.isInitialized) {
+      SoLoud.instance.shutdown();
+    }
```

Comment:
[nit] Is it really needed to shut down the player? Okay to leave for now, but probably not good if this was a thing developers would need to do.

Reply:
I just wanted to have a fresh player created, also to test the new code. I was also thinking of splitting the tabs into separate .dart codes.
Without this, it is needed to check if the player is initialized and if yes dispose all the sounds. It's just my laziness I suppose

Reply:
I have rearranged the initialization.
The problem I was facing and that I didn't know, was that when you change tabs from A to B, the `B.initState()` is called before `A.dispose()`. And I was trying to shut down the player in the `A.dispose()`.